falcon_kit/bash.py

```diff
@@ -294,6 +294,8 @@ def script_run_consensus(config, db_fn, las_fn, out_file_bfn):
     params.update(locals())
     if config["falcon_sense_skip_contained"]:
         run_consensus = """LA4Falcon -H$CUTOFF -fso {db_fn} {las_fn} | """
+    elif config["falcon_greedy"]:
```

Comment:
Hmmm. Not the best name. The whole program is called 'falcon', so that part adds nothing. (Above we have `falcon_sense_skip_contained` because Jason sometimes calls fc_consensus "falcon-sense".) Maybe `falcon_sense_greedy`?

Also, this becomes exclusive of the `skip_contained` option. Is that intended? I think Jason said he no longer uses skip-contained, but I'm not sure.